src/Neves/Events/TransactionalDispatcher.php

```diff
@@ -146,6 +146,12 @@ protected function addPendingEvent($connection, $event, $payload)
         $connectionId = $connection->getName();
         $transactionLevel = $this->transactionLevel;
 
+        // When working with multiple connections, we might come across
```

Comment:
I see this proposal as defensive. This is just a possible use-case, so this comment can be removed.